src/tools/ScenarioMeasurement/MemoryConsumption/AndroidMemoryParser.cs

```diff
@@ -0,0 +1,84 @@
+using Microsoft.Diagnostics.Tracing;
+using Reporting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace ScenarioMeasurement
+{
+
+    /// <summary>
+    /// This is a custom parser that does not enable any profiling. Instead, it relies on being passed files collected
+    /// off the test machine (usually on a device.) 
+    /// </summary>
+    class AndroidMemoryParser : IParser
+    {
+        public void EnableKernelProvider(ITraceSession kernel)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void EnableUserProviders(ITraceSession user)
+        {
+            throw new NotImplementedException();
+        }
+
+        public IEnumerable<Counter> Parse(string mergeTraceFile, string processName, IList<int> pids, string commandLine)
+        {
+            var pssMins = new List<double>();
+            var pssAvgs = new List<double>();
+            var pssMaxs = new List<double>();
+            var ussMins = new List<double>();
+            var ussAvgs = new List<double>();
+            var ussMaxs = new List<double>();
+            var rssMins = new List<double>();
+            var rssAvgs = new List<double>();
+            var rssMaxs = new List<double>();
+            var captureCounts = new List<double>();
+            // 52MB-52MB-52MB/45MB-45MB-45MB/138MB-138MB-138MB over 1
+            // PSS: min {pss[0]}, avg {pss[1]}, max {pss[2]}; USS: min {uss[0]}, avg {uss[1]}, max {uss[2]}; RSS: min {rss[0]}, avg {rss[1]}, max {rss[2]}; Number: {CaptureNumber}
```

Comment:
If the format of the output from the Android devices changes to we have to update both the regex here as well as the one in `runner.py`? If possible I would like to make that not true so we can limit the complexity of changes here in the future.